scripts/verify-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+echo "running tests..."
+gotestcover -race -covermode=count -coverprofile=cover.out $(go list ./... | grep -v /vendor/)
+
+echo ""
+go tool cover -func cover.out
```

Comment:
Much nicer, thanks.

I think this script & verify-preflight.sh should just be embedded directly into the Makefile.